lib/matplotlib/tests/test_axes.py

```diff
@@ -351,7 +349,7 @@ def test_shaped_data():
     plt.plot(y2)
 
     plt.subplot(413)
-    assert_raises(ValueError, plt.plot, (y1, y2))
+    pytest.raises(ValueError, plt.plot(y1, y2))
```

Comment:
I misread the "Assertions about expected exceptions" section in https://pytest.org/latest/assert.html , so yes, if I just replaced assert_raises with pytest.raise, I would have been ok.  I'll change it back, will probably revert the bigger files that I changed.